### DISCUSSION TITLE
Screen Sharing OnlyM to Zoom during Hybrid Meetings

### DISCUSSION THREAD

**Discussion by urspaghett** [AntonyCorbett/OnlyM#445] [Q&A]
Hello All!

Currently for hybrid meetings we are using the following setup:

One Windows PC, connected to the two TV monitors. 

The PC hosts the Zoom meeting, and is used for media playback through JW Library. 

We use OnlyM to place third party images on the TV Monitors, but have found that if we attempt to Screen Share OnlyM as well, the Zoom controls are sent to the TV's, disrupting the meeting. I've fiddled endlessly with the settings and haven't found a way to make this work without some sort of disruption. 

Curious if anyone has found a different way to work around this that's simpler than our current process: screen sharing the third party image from a photo viewer application simultaneously as we play the image on the monitors through OnlyM.

And if the dev has any insight, it would be much appreciated!

**Comment by AntonyCorbett** [comment]
You should share the second screen, not the application.

**Reply by urspaghett** [reply]
Hi Antony, 

I have tried that. When this is done, with our setup, the Zoom window gets sent to the monitors, forcing us to the drag it back to the main PC.

**Reply by mvpapen** [reply]
Hi there. We had very similar results. As soon your activate dual screen in Zoom , the share function is very unpredictable.
 We solved it by using a capture card to capture hdmi out to 2 monitor. the capture card is connected  via usb to the pc and shows as 2 camera in Zoom.
you can now toggle between the 2 cameras ( ALT+N) instead of sharing. Works great. We also use Focus mode to encourage the friends to stay in speaker view. After the meeting we turn off "Focus Mode"

**Reply by joelgray76** [reply]
> You should share the second screen, not the application.

Hi brother, is there any way I can contact you directly?

**Comment by joelgray76** [comment]
> Hello All!
> 
> Currently for hybrid meetings we are using the following setup:
> 
> One Windows PC, connected to the two TV monitors.
> 
> The PC hosts the Zoom meeting, and is used for media playback through JW Library.
> 
> We use OnlyM to place third party images on the TV Monitors, but have found that if we attempt to Screen Share OnlyM as well, the Zoom controls are sent to the TV's, disrupting the meeting. I've fiddled endlessly with the settings and haven't found a way to make this work without some sort of disruption.
> 
> Curious if anyone has found a different way to work around this that's simpler than our current process: screen sharing the third party image from a photo viewer application simultaneously as we play the image on the monitors through OnlyM.
> 
> And if the dev has any insight, it would be much appreciated!

Hi there, can you explain your issue a little more? What exactly are you having problems with? 
I use OBS studio in the few kingdom halls I have setup systems in. We have a great system that works really well.

**Comment by HakanLG** [comment]
Hello, how is it going with your problem?

If it's still there, try to move or drag the Zoom controls back from the TV's  to the main screen.
Zoom will remember this place next time you will Share Screen.

**Comment by mvpapen** [comment]
You can drag the zoom controls back to your main screen.
Next time you use it it should stay on your main screen.
If not, test and move controls to main screen ,than close zoom while
pressing the shift key.
It tells windows to remember your windows positions.
Hope that helps.
